theano/tensor/tests/test_opt.py

```diff
@@ -4658,7 +4658,7 @@ def test_consecutive(self):
         f(dx)
         topo = f.maker.fgraph.toposort()
         assert len(topo) == 1
-        assert isinstance(topo[0].op.scalar_op, scal.basic.Cast)
+        assert isinstance(topo[0].op, T.Elemwise)
```

Comment:
`assert isinstance(topo[0].op, T.Elemwise)` is the current line in the Theano repo. I thought it needed changing, and then put it back when my own tests started failing. I'll change it to `scal.basic.Cast` again.